changelog.d/20221202_004331_kevin.matthes_lexer_curly_squash.rst

```diff
@@ -0,0 +1,6 @@
+.. _#1547: https://github.com/fox0430/moe/pull/1547
```

Comment:
@fox0430

When running the workflow added by #1544, this CHANGELOG fragment will be automatically shelved in correctly into the assembled CHANGELOG.

You can test the behaviour of the workflow by creating a new branch and committing the release workflow's result thereto.  After inspecting the results, you can delete the test branch.